add team akaringular first blog

StlMaris123 opened the pull request:
This pr is our first blog post

StlMaris123 committed:
add team akaringular first blog

v-thakkar commented on blog/_posts/2018-07-13-team-akaringular.md:
```diff
@@ -0,0 +1,36 @@
+---
+**title:** New Tech Journey With RGSoC!
```
Hi @StlMaris123 ! Thanks for your PR. There is no need of '**' in the parameters here. Please remove them.

v-thakkar commented on blog/_posts/2018-07-13-team-akaringular.md:
```diff
@@ -0,0 +1,36 @@
+**current:** blog
+**author:** Rachael Kiteme
+**twitter:** rachaelkiteme
+**categories:**  news
```
This should be under 'student-posts' category. Please check the format [from other files] to declare categories parameter as well.

v-thakkar commented on blog/_posts/2018-07-13-team-akaringular.md:
```diff
@@ -0,0 +1,36 @@
+
+
+#:confetti_ball::computer:New Tech Journey With RGSoC :computer::confetti_ball:
+![New Dawn!](/img/blog/2018/akaringular_begin.jpeg)
```
Use 'image-class' here to add image credits. You can check other posts in the repo to know how to add image credits. Also, if the image size is small then make sure that the image is in the center.

v-thakkar commented on blog/_posts/2018-07-13-team-akaringular.md:
```diff
@@ -0,0 +1,36 @@
+Sometimes in life we don’t realize how one simple action, one simple mistake affects our life until it pushes us to a wall, thats when it hits us hard that we have been heading to wrong direction. Imagine finishing four years in university calling yourself a developer because you knew how to program. Hackathons didn’t pass me by, Ruby meet ups, Android app development classes and tech conferences ...
+Not until I finished university and had to put my skills on practice just to realize the only code I could write comfortably was “Hello World” but in different programming languages, that’s when it downed on me that I had been carrying the name “Developer” on my back.
+
+![Sad Truth!](/img/blog/2018/akaringular_languages.jpeg)
```
Same comment as above. Please add image credits.

v-thakkar commented on blog/_posts/2018-07-13-team-akaringular.md:
```diff
@@ -0,0 +1,36 @@
+![New Dawn!](/img/blog/2018/akaringular_begin.jpeg)
+
+Sometimes in life we don’t realize how one simple action, one simple mistake affects our life until it pushes us to a wall, thats when it hits us hard that we have been heading to wrong direction. Imagine finishing four years in university calling yourself a developer because you knew how to program. Hackathons didn’t pass me by, Ruby meet ups, Android app development classes and tech conferences ...
+Not until I finished university and had to put my skills on practice just to realize the only code I could write comfortably was “Hello World” but in different programming languages, that’s when it downed on me that I had been carrying the name “Developer” on my back.
```
Typo here: 'downed' -> 'dawned'

v-thakkar commented on blog/_posts/2018-07-13-team-akaringular.md:
```diff
@@ -0,0 +1,36 @@
+
+>"If we wait for the moment when everything, absolutely everything is ready, we shall never begin” by Ivan Turgenev
+
+The quote!!
```
I think this sentence is unnecessary. May be remove it.

v-thakkar commented on blog/_posts/2018-07-13-team-akaringular.md:
```diff
@@ -0,0 +1,36 @@
+
+The quote!!
+
+Yes, we had to try it out RGSoC internship program 2018, never doubting my abilities anymore and luckily the trial turned to a surprise video, it was a shock at fast, took a minute to read Vaishali Thakkar tough question who’s answer was just a huge Hurrray!!!!!! shout out.We made it!:tada: :tada: :tada: :tada:
```
Add blank space before sentence 'We made it'. :)

v-thakkar commented on blog/_posts/2018-07-13-team-akaringular.md:
```diff
@@ -0,0 +1,36 @@
+
+Yes, we had to try it out RGSoC internship program 2018, never doubting my abilities anymore and luckily the trial turned to a surprise video, it was a shock at fast, took a minute to read Vaishali Thakkar tough question who’s answer was just a huge Hurrray!!!!!! shout out.We made it!:tada: :tada: :tada: :tada:
+
+Tik Tok, The clock was ticking and July was fast approaching, and only two days to the commencement of the internship my teammate StellaMaris Njage lost her Laptop :sob: :sob:. That was really sad, but this didn’t stop us from getting started. What are friends for? stella ireri helped her with one. Yeah, she got set lucky enough to have our kick off call on time together and our new tech journey  ...
```
I think you should make 'S' capital for 'stella ireri'.

v-thakkar commented on blog/_posts/2018-07-13-team-akaringular.md:
```diff
@@ -0,0 +1,36 @@
+Tik Tok, The clock was ticking and July was fast approaching, and only two days to the commencement of the internship my teammate StellaMaris Njage lost her Laptop :sob: :sob:. That was really sad, but this didn’t stop us from getting started. What are friends for? stella ireri helped her with one. Yeah, she got set lucky enough to have our kick off call on time together and our new tech journey  ...
+
+
+I love the journey so far, RGSoC has instilled in me coding schedule, making sure that I learn something daily, such volunteering mentors and supervisors to help me when stack and making sure I work with a close partner, someone am used to and always close to. I cant regret being here because am growing drastically each day as a tech lady and career wise. Am ready enough to work with the communit ...
```
Bunch of minor issues. Change 'stack->I'm stuck' , 'with a close partner->closely with a partner', 'grown to->grown into'.

StlMaris123 committed:
Update 2018-07-13-team-akaringular.md

Implement the requested changes

v-thakkar commented on blog/_posts/2018-07-13-team-akaringular.md:
```diff
@@ -0,0 +1,36 @@
+---
+title: New Tech Journey With RGSoC!
+layout:post
```
Hi @StlMaris123 ! You need to add one blank space between ':' and text here and in other parameters below.

v-thakkar commented on blog/_posts/2018-07-13-team-akaringular.md:
```diff
@@ -0,0 +1,36 @@
+current: blog
+author:Rachael Kiteme
+twitter:rachaelkiteme
+categories: student-posts
```
For the consistency purpose, please declare this as ' - student-posts' in the next line. You can check other files for the example.

StlMaris123 committed:
Update 2018-07-13-team-akaringular.md

Implemented changes requested

StlMaris123 committed:
rewrite blog as a team

StlMaris123 committed:
Update 2018-07-13-team-akaringular.md

Add headings

v-thakkar commented:
Thanks for all the changes @StlMaris123. The content of the post looks much better. And I'm happy with it looking more like a team blog post. :)

Commit:
Final changes to Team Akaringular's first blog post

- Renamed the md file to current date
- Renames images files
- Added legends to the images (it only had credits)
- Cleaned up emojis (they weren't working as they were mostly text in between ":")
- General formatting
